veocean-data-farming/faq.md

```diff
@@ -0,0 +1,121 @@
+</details>
+
+## veOCEAN FAQ
+<details>
+<summary>What is the official formula for the Linear Decay?</summary>
+
+The Linear Decay formula for veOCEAN can be expressed as follows in python.  
+```python
+FOUR_YEARS = 60 * 60 * 24 * 7 * 52
+
+veOcean_balance = OCEAN_amount_locked * (your_unlock_timestamp — current_unix_timestamp ) / FOUR_YEARS
+```
+
+To learn more about systems driving veOCEAN and Data Farming, please [visit our df-py github repository](https://github.com/oceanprotocol/df-py).
+</details>
+
+## Staking FAQs
+<details>
+<summary>What about passive stakers — people who just want to stake in one place and be done?</summary>
+
+Earnings are passive by default
+</details>
+
+<details>
+<summary>What about active stakers — people who want to do extra work and get rewarded?</summary>
+
+Ot works. Half the DF revenue goes to veOCEAN stake that users can allocate. Allocate well → more $$
+</details>
+
+## Third-Party Market FAQs
+<details>
+<summary>What do you think could be the monetization / incentive options for third party dApps and Data Markets that build on top of OCEAN?</summary>
+
+People pay fees on fixed-rate exchange and in consume. 3rd party markets can get a cut of both. More info at Ocean docs.
+</details>
+
+<details>
+<summary>Staking veOCEAN will be an important part in curating the most valuable datasets. Which earning opportunities will stakers have other than DF and fees generated from consume transactions?</summary>
+
+veOCEAN holders’ earning potential via fees and DF on their own could be quite substantial, eg given that a large portion of OCEAN supply is directed towards DF. No need for more ways per se. There won’t be a fee on locking OCEAN for veOCEAN, because adding that code would have meant changes to veCRV contracts, something we’re not doing for security reasons. Re ALGA or other apps making money on facilitating veOCEAN: there are several ways, the best way to get inspiration is to look at projects on top of base ve tokens, such as Convex, Aura, Redacted Cartel, and more.
+</details>
+
+## Pricing FAQs
+<details>
+<summary>In this scheme, can people stake on fixed-price datasets?</summary>
+
+Yes. They allocate their veOCEAN to datasets. Then DF rewards follow the usual DF formula: DCV * veOCEAN stake.
+</details>
+
+<details>
+<summary>Does this work for other pricing schemes?</summary>
+
+Yes, from the get-go! It doesn’t matter how data is priced, this works for all schemes.
+</details>
+
+<details>
+<summary>With pools getting wound down in Ocean Market, will fixed-price be its only pricing scheme?</summary>
+
+For now in Ocean Market, yes. However people can price datatokens however they like, leveraging whatever DeFi tools they like. For example. do an IDO via Liquidity Bootstrapping Pool, ending up in an unmodified Balancer AMM. And we will continue listening to the community, to understand best where to focus our efforts.
```

Comment:
Remove. This is historical.

Reply:
Removed... although I think the following part is still useful: ```However people can price datatokens however they like, leveraging whatever DeFi tools they like. For example. do an IDO via Liquidity Bootstrapping Pool, ending up in an unmodified Balancer AMM. And we will continue listening to the community, to understand best where to focus our efforts.```